Jetcaster/app/src/main/java/com/example/jetcaster/ui/JetcasterApp.kt

```diff
@@ -16,11 +16,55 @@
 
 package com.example.jetcaster.ui
 
+import android.content.Context
+import android.net.ConnectivityManager
+import androidx.compose.material.AlertDialog
+import androidx.compose.material.Text
+import androidx.compose.material.TextButton
 import androidx.compose.runtime.Composable
+import androidx.compose.runtime.mutableStateOf
+import androidx.compose.runtime.remember
+import androidx.compose.ui.platform.AmbientContext
+import androidx.compose.ui.res.stringResource
+import com.example.jetcaster.R
 import com.example.jetcaster.ui.home.Home
 
+
 @Composable
 fun JetcasterApp() {
+    val context = AmbientContext.current
+    val isOnline = remember { mutableStateOf(checkIfOnline(context)) }
```

Comment:
using `by` avoids having to use the `.value` below which reads better

```suggestion
    var isOnline by remember { mutableStateOf(checkIfOnline(context)) }
```